gnosis/safe-react
# fix: checksum Safe App transaction addresses

## What it solves
Resolves #3736

## How this PR fixes it
Transactions sent via the Safe App `communicator` have their addresses checksummed.

## How to test it
Open a contract interaction modal, i.e. via interacting with a Safe App, and observe the checksummed address.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/160637375-4ba13194-9e25-47f3-ac07-4b194402dff3.png)
